cmd/skaffold/app/cmd/render.go

```diff
@@ -47,7 +46,8 @@ func NewCmdRender() *cobra.Command {
 			{Value: &showBuild, Name: "loud", DefValue: false, Usage: "Show the build logs and output", IsEnum: true},
 			{Value: &renderFromBuildOutputFile, Name: "build-artifacts", Shorthand: "a", Usage: "File containing build result from a previous 'skaffold build --file-output'"},
 			{Value: &offline, Name: "offline", DefValue: false, Usage: `Do not connect to Kubernetes API server for manifest creation and validation. This is helpful when no Kubernetes cluster is available (e.g. GitOps model). No metadata.namespace attribute is injected in this case - the manifest content does not get changed.`, IsEnum: true},
-			{Value: &renderOutputPath, Name: "output", Shorthand: "o", DefValue: "", Usage: "file to write rendered manifests to"},
+			// This "--output" flag is equivalent to the "--render-output" flag defined in SkaffoldOpts due to user convenience.
```

Comment:
_Replacement for `run --render-output` flag_

The `--render-output` flag is deprecated.

Reply:
Done.